src/app/shared/event-log/event-log-table.component.spec.ts

```diff
@@ -102,7 +102,7 @@ describe('EventLogTableComponent', () => {
     let secondEvent = EVENTS[1];
 
     expect(secondRowCells[COL_EVENT].nativeElement.textContent).toBe(secondEvent.event_name);
-    expect(secondRowCells[COL_DATE].nativeElement.textContent).toBe('9 May 2017, 4:07:03 PM');
+    expect(secondRowCells[COL_DATE].nativeElement.textContent).toBe('9 May 2017, 4:07:03 PM UTC: 9 May 2017, 4:07:03 PM');
```

Comment:
Why are we changing this?

Reply:
@MKaramath added a local date modifier and a tooltip for the event details component, however this table seems to have been forgotten. Which leads different time displayed in log table and detail. This is just for consistency.

Reply:
👍